docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+---
+# Profiles in Visual Studio Code
+
+Visual Studio Code has 100s of [settings](/docs/getstarted/settings.md), 1000s of [extensions](/docs/editor/extension-marketplace.md), and unnumerable ways to adjust the UI layout to customize the editor. VS Code **Profiles** let you create sets of customizations and quickly switch between them or share them with others. This topic explains how to create, modify, export, and import profiles.
+
+## Create a Profile
+
+VS Code treats your current configuration as the **Default Profile**. As you modify settings, install extensions, or open and close views and panels, these customizations are tracked in the Default Profile.
+
+To create a new profile, you can use the **File** > **Preferences** > **Profiles** > **Create Profile** menu item (**Code** > > **Preferences** > **Profiles** > **Create Profile** on macOS).
+
+![Create Profile command](images/profiles/create-profile.png)
+
+You can create a new profile based on the current profile (**Profiles: Create from Current Profiles**) or create an Empty Profile. An Empty Profile includes no user customizations (settings, extensions, snippets, etc.) but with the UI state (view layouts and sizing) preserved.
+
+You can also access the Profile command menu via the **Manage** gear button in the lower right of the Activity bar.
+
+![Create Profile command via the Manage button in the Activity bar](images/profiles/create-profile-via-manage.png)
+
+The **Profiles: Create Profile** command is also available in the Command Palette (`kb(workbench.action.showCommands)`).
+
+Once you choose whether to create a new profile based on the current profile or an empty profile, you are prompted to enter a name for the new profile.
+
+### Check the current profile
+
+The current profile name is displayed in several places in the VS Code UI:
+
+* Title bar
+* **File** > **Preferences** > **Profiles**
+* **Manage** gear button hover
+
+If you are still using the Default Profile, no profile name is displayed.
+
+The **Manage** gear button displays a badge with the first two letters of the active profile so you can quickly check which profile you are running.
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
+
+### Workspace associations
+
+When you select create or select an existing profile, it is associated with the current workspace and whenever you open that folder, the workspace's profile is active. If you open another folder, the profile either goes back to the Default Profile if no profile is set, or switches to that folder's associated profile.
```

Comment:
"If you open another folder, the profile either goes back to the Default Profile if no profile is set"

This is not correct, we will use the profile that you last used in the previous folder. Right? @sandy081

Reply:
Yes, thats correct. A newly opened folders takes the profile of the previous window.

Reply:
Hmm, sometimes works that way but not if you've explicitly switched back to the Default Profile, which I thought cleared the profile association.
I'll update the topic and play around with this a bit more

Reply:
Added a paragraph